src/hll.c

```diff
@@ -323,7 +328,11 @@ FunctionOid(const char *schemaName, const char *functionName, int argumentCount,
 	Oid functionOid = InvalidOid;
 
 	char *qualifiedFunctionName = quote_qualified_identifier(schemaName, functionName);
-	List *qualifiedFunctionNameList = stringToQualifiedNameList(qualifiedFunctionName);
+	List *qualifiedFunctionNameList = stringToQualifiedNameList(qualifiedFunctionName
```

Comment:
Could we remove  stringToQualifiedNameList alltogether replacing it with the following?

  `qualifiedFunctionNameList = list_make1(makeString(qualifiedFunctionName));`

It looks simpler this way.